packages/instant/src/components/buy_order_progress.tsx

```diff
@@ -1,3 +1,4 @@
+import * as _ from 'lodash';
```

Comment:
is this fixing a regression?

Reply:
hmmm I saw this but assumed running `yarn lint --fix` would fix it. Clearly it doesn't catch this case.